node_modules/oae-content/lib/api.js

```diff
@@ -939,8 +947,16 @@ var _canShareContent = function(ctx, contentObj, callback) {
             // If we can interact with the item, we can always share it
             return callback(null, true);
         } else if (contentObj.visibility === AuthzConstants.visibility.PRIVATE) {
-            // If the content is private, we need to be a manager to share it
-            return AuthzAPI.hasRole(user.id, contentObj.id, ContentConstants.roles.MANAGER, callback);
+            // If the content is private, we need to be a manager or editor to share it
+            return AuthzAPI.getAllRoles(ctx.user().id, contentObj.id, function(err, roles) {
+                if (err) {
+                    callback(err);
+                }
+                var _isEditorOrManager = _.some(roles, function(role) {
```

Comment:
Doesn't need to be prefixed with a `_` because it isn't an internal parameter or method, it's just a regular variable.